script.js

```
@@ -1,35 +1,70 @@
+    Number(localStorage.getItem("currentTarget")) || currentTarget;
+
+  currentConsumed =
+    Number(localStorage.getItem("currentConsumed")) || 0;
+
+  document.getElementById("target-val").innerText = currentTarget;
+
+  const savedLogs = JSON.parse(localStorage.getItem("logs")) || [];
+  const list = document.getElementById("log-list");
+  list.innerHTML = "";
+
+  savedLogs.forEach((log) => restoreLog(log));
```

Biome (2.3.13)

[error] This callback passed to forEach() iterable method should not return a value.

Either remove this return or remove the returned value.


(lint/suspicious/useIterableCallbackReturn)